bin/console

```diff
@@ -7,7 +7,7 @@ require "ingram_micro"
 # with your gem easier. You can also use a different console, if you like.
 
 # (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
+require "pry"
```

Comment:
sorry, I meant to comment it out. it isn't in the production gemspec so we don't want it in an executable path.